src/iccpd/src/mlacp_sync_update.c

```diff
@@ -523,42 +772,73 @@ int mlacp_fsm_update_arp_entry(struct CSM* csm, struct ARPMsg *arp_entry)
                     }
                     else
                     {
+                        ln = __LINE__;
                         continue;
                     }
                 }
             }
         }
     }
 
+    if(my_ip_arp_flag)
+    {
+        ICCPD_LOG_DEBUG(__FUNCTION__," ignoring ARP sync for self ip %s ", show_ip_str(arp_entry->ipv4_addr));
+        return 0;
+    }
+
     /* set dynamic ARP*/
     if (set_arp_flag == 1)
     {
+        if (arp_entry->flag & NEIGH_SYNC_FLAG_SELF_IP)
+        {
+            permanent_neigh = 1;
+        }
+
         if (arp_entry->op_type == NEIGH_SYNC_ADD)
         {
-            if (iccp_netlink_neighbor_request(AF_INET, (uint8_t *)&arp_entry->ipv4_addr, 1, arp_entry->mac_addr, arp_entry->ifname) < 0)
+            err = iccp_netlink_neighbor_request(AF_INET, (uint8_t *)&arp_entry->ipv4_addr, 1, arp_entry->mac_addr, arp_entry->ifname, permanent_neigh, 8);
+            if (err < 0)
             {
-                ICCPD_LOG_WARN(__FUNCTION__, "ARP add failure for %s %s %s",
-                                arp_entry->ifname, show_ip_str(arp_entry->ipv4_addr), mac_str);
-                return MCLAG_ERROR;
+                if (err != ICCP_NLE_SEQ_MISMATCH) {
+                    ICCPD_LOG_NOTICE(__FUNCTION__, "ARP add failure for %s %s %s, status %d",
```

Comment:
An error should be logged as ERROR and not NOTICE right?

Reply:
Thanks, Done.!